docs/docs/api/websocket.md

```diff
@@ -81,3 +82,17 @@ field informs the client of the kind of update that is encoded in the payload.
   }
 }
 ```
+
+### Tag
+
+```json5
+{
+  "type": "tag",
```

Comment:
So for my understanding: For now we don't handle tag deletions?

Reply:
No, the reason why I didn't do this is that right now "deleting tags" is not though off so well as it doesn't cascade to conversations (and maybe we should never do that. I guess we should discuss this in a separate issue though)